packages/bpk-docs/src/components/DocsPageBuilder/ComponentVideos.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import BpkText from 'bpk-component-text';
+import BpkPanel from 'bpk-component-panel';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './ComponentVideos.scss';
+
+const getClassName = cssModules(STYLES);
+
+let counter = 0;
+const generateId = (title = '') => {
+  counter += 1;
+  return `${title.toLowerCase().replace(' ', '-')}-${counter}`;
+};
+
+const ComponentVideo = ({ title, subText, src, ...rest }) => {
+  const subTextId = generateId(title);
+
+  return (
+    <div key={title} className={getClassName('bpkdocs-component-videos__item')}>
+      <dt className={getClassName('bpkdocs-component-videos__item-title')}>
+        <BpkText tagName="h3">{title}</BpkText>
+      </dt>
+      <dd
+        className={getClassName(
+          'bpkdocs-component-videos__item-video-container',
+        )}
+      >
+        <BpkPanel>
+          <video
+            loop
+            controls
+            className={getClassName('bpkdocs-component-videos__item-video')}
+            autoPlay
```

Comment:
Can we also add `playsInline` and `muted` too?

`playsInline` will mean that on small screen devices the video plays inline (without this, on iPhones the video *only* play in full screen).

Reply:
You're awesome! yes will do!